DeepMenu/Main/ContentView.swift

```diff
@@ -14,9 +14,9 @@ struct ContentViews {
 struct ContentView: View {
 
     let hub1 = MuHub([.lower, .right])
-    let hub2 = MuHub([.lower, .left ])
-    let hub3 = MuHub([.upper, .right])
-    let hub4 = MuHub([.upper, .left ])
+//    let hub2 = MuHub([.lower, .left ])
+//    let hub3 = MuHub([.upper, .right])
+//    let hub4 = MuHub([.upper, .left ])
```

Comment:
I still think these aren't useful

Reply:
What do you mean? This was for testing each corner, which may reverse to pod order. Perhaps put in UITests?

Reply:
I mean adding back in the `//`  ...I had taken them out because they're just the declarations, the comment slashes a littler further down are sufficient for turning on/off the corners at compile time

Reply:
Mostly what I'm saying is that when commenting out these lines, when you want to turn a corner back on, you have to edit two lines instead of just one